packages/components/addon/components/hds/alert/index.hbs

```diff
@@ -0,0 +1,40 @@
+<div class={{this.classNames}} ...attributes role="alert">
+  {{#if this.icon}}
+    <div class="hds-alert__icon">
+      <FlightIcon @name={{this.icon}} @size={{this.iconSize}} @stretched={{true}} @isInlineBlock={{false}} />
+    </div>
+  {{/if}}
+
+  <div class="hds-alert__content">
+    <div class="hds-alert__text">
+      {{#if @title}}
+        <div class="hds-alert__title">{{@title}}</div>
+      {{/if}}
+      {{#if @description}}
+        <div class="hds-alert__description">{{html-safe @description}}</div>
+      {{/if}}
+    </div>
+
+    {{! IMPORTANT: here we have to rely on handlebar's "squishies" to remove the extra whitespace added by Ember (we're using ':empty' in CSS to hide it when it's empty!) }}
+    {{! Notice: this needs to be done also on the consumer side too, unfortunately ¯\_(ツ)_/¯ }}
+    {{! See here: https://handlebarsjs.com/guide/expressions.html#whitespace-control }}
+    {{~#if (has-block "actions")~}}
+      <div class="hds-alert__actions">
+        {{~yield
+          (hash
+            Button=(component "hds/button" size="small")
+            Link::Standalone=(component "hds/link/standalone" size="small")
+            LinkTo::Standalone=(component "hds/link-to/standalone" size="small")
+          )
+          to="actions"
+        ~}}
+      </div>
+    {{~/if~}}
+  </div>
+
+  {{#if this.onClose}}
+    <button class="hds-alert__dismiss" type="button" {{on "click" this.onClose}}>
```

Comment:
[issue, blocker] this is missing the `aria-label` attribute.

Reply:
done